README.md

```diff
@@ -1,22 +1,58 @@
 ViperMonkey
 ===========
 
+This is a fork of Philippe Lagadec's (http://www.decalage.info) ViperMonkey.
+
 ViperMonkey is a VBA Emulation engine written in Python, designed to analyze
 and deobfuscate malicious VBA Macros contained in Microsoft Office files
 (Word, Excel, PowerPoint, Publisher, etc).
 
-See my article "[Using VBA Emulation to Analyze Obfuscated Macros](http://decalage.info/vba_emulation)",
+See Philippe's article "[Using VBA Emulation to Analyze Obfuscated Macros](http://decalage.info/vba_emulation)",
 for real-life examples of malware deobfucation with ViperMonkey.
 
 **DISCLAIMER**:
-- ViperMonkey is a *very* incomplete and experimental VBA Engine. For now it will NOT
-handle most real-life macros without errors.
-- VBA parsing and emulation is *extremely* slow for now.
+- ViperMonkey is an experimental VBA Engine targeted at analyzing maldocs. It works on some but not all maldocs. 
+- VBA parsing and emulation is *extremely* slow for now (see the speedup section for how to improve the speed).
 - VBA Emulation is hard and complex, because of all the features of the VBA language, of Microsoft
 Office applications, and all the DLLs and ActiveX objects that can be called from VBA.
 - This open-source project is only developed on my scarce spare time, so do not expect
 miracles. Any help from you will be very appreciated!
 
+**oletools Version**
+
+ViperMonkey requires the most recent version of [oletools]
+(https://github.com/decalage2/oletools), not the version downloaded
+pip. Make sure to either install the most recent oletools version by
+running the oletools 'setup.py install' or by making sure the most
```

Comment:
Or even better, see https://github.com/decalage2/oletools/wiki/Install#how-to-install-the-latest-development-version for a pip one-liner.